skin.json

```diff
@@ -129,7 +129,16 @@
   "closedCaptionOptions": {
     "defaultEnabled": true,
     "defaultLanguage": "en",
-    "availableLanguages": []
+    "availableLanguages": [],
+    "defaultTextColor": "white",
```

Comment:
This is interesting.  @wunnath  we should talk about if this affects our SDKs FCC Closed Captions styling stuff. Overall i think this is fine here